src/content/en/updates/2017/09/auto-play-policy-changes.md

```diff
@@ -0,0 +1,154 @@
+where video is the primary content type. When it is high enough, media playback
+is allowed to auto-play on desktop only.
+
+Note that you can check out user's MEI at the <i>chrome://media-engagement/</i>
+internal page in Chrome 62.
+
+<figure>
+  <img src="/web/updates/images/2017/09/media-engagement.png"
+       alt="Screenshot of the chrome://media-engagement page">
+  <figcaption>
+    <b>Figure 2.</b>
+    TODO: Screenshot of the <i>chrome://media-engagement</i> internal page
+  </figcaption>
+</figure>
+
+### Iframe delegation {: #iframe }
+
+Once an origin has received auto-play permission, it can delegate that
+permission to iframes via a new HTML attribute:
+
+    <iframe src="myvideo.html" gesture="media">
+
+Without iframe delegation, videos will not be able to autoplay with sound.
+
+### Example scenarios
+
+<b>Example 1:</b> Every time a user visits <i>VideoSubscriptionSite.com</i> on their
+laptop they watch a TV show or a movie. As their Media Engagement score is
+high, auto-play is allowed.
+
+<b>Example 2:</b> <i>GlobalNewsSite.com</i> has both text and video content.
+Most users go to the site for text content and watch videos only occasionally.
+Users' Media Engagement score is low, so auto-play wouldn't be allowed if a user
+navigates directly from a social media page or search. 
+
+<b>Example 3:</b> <i>LocalNewsSite.com</i> has both text and video content.
+Most people enter the site through the homepage and then click on the news
+articles. Autoplay on the news article pages would be allowed because of user
+interaction with the domain. However, care should be taken to make sure users
+aren't surprised by auto-playing content. 
+
+<b>Example 4:</b> <i>MyMovieReviewBlog.com</i> embeds an iframe with a movie
+trailer to go along with their review. The user interacted with the domain to
+get to the specific blog, so auto-play is allowed. However, the blog needs to
+explicitly delegate that privilege to the iframe in order for the content to
+auto-play.
+
+## Best Practises for Web Developers {: #best-practises }
+
+Here's the one thing to remember: Don't ever assume a video will play, and
+don't show a pause button from the start. It is so important that I'm going
+to write one more time below for those who simply skim through that post.
+
+Key Point: Don't ever assume a video will play, and don't show a pause button
+from the start.
+
+You should always look at the [Promise] returned by the play function to see if
+it was rejected:
+
+    var promise = document.querySelector('video').play();
+    
+    if (promise !== undefined) {
+      promise.then(_ => {
+        // Auto-play started!
+      }).catch(error => {
+        // Auto-play was prevented.
+        // Show a "Play" button so that user can start playback.
+      });
+    }
+
+Warning: Don't play interstitial ads without showing any
+media controls as they may not auto-play and user will have no way of
+starting playback.
```

Comment:
Maybe recommend muted autoplay with unmute button?

Reply:
Isn't what is just below?